sql/wsrep_binlog.cc

```diff
@@ -430,6 +430,8 @@ void wsrep_register_for_group_commit(THD *thd) {
 }
 
 bool wsrep_implicit_transaction(THD *thd) {
+  if (!thd) return false;
```

Comment:
I think we already have this check in the `release-8.0.33` branch. Please revert this check and change the target branch of this PR to `release-8.0.33`.